src/content/enterprise/eks.mdx

```diff
@@ -28,9 +28,10 @@ This guide will assume that your domain is registered in [Amazon Route53](https:
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow Amazon's [cluster creation guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+Okteto supports Kubernetes versions 1.20 through 1.22.
```

Comment:
@pchico83 do we want to say 1.21 given the issue with volume snapshots?

Reply:
@rberrelleza we have tested the rest of the functionality and everything was green. Also, I think it's not about the kubernetes version. It's related to the CSI driver version that you install in the EKS cluster.
Therefore, I would keep 1.22 as supported